dynamic-demo-plugin/yarn.lock

```diff
@@ -81,8 +81,8 @@
 "@openshift-console/dynamic-plugin-sdk@file:../frontend/packages/console-dynamic-plugin-sdk/dist/core":
   version "0.0.0-fixed"
   dependencies:
-    "@patternfly/react-core" "4.168.8"
-    "@patternfly/react-table" "4.37.8"
+    "@patternfly/react-core" "4.175.4"
```

Comment:
Do we need to update PatternFly as part of this fix?

Reply:
Nope, but yarn thought it was a good idea to upgrade. I can definitely drop the lock file change, I saw it as I was cherry picking. We probably should look at a way to lock down our dependencies if we don't want auto upgrades for z-streams.

Reply:
Hm, I don't think `yarn install` would update the version in the lockfile... I'm not sure what happened, but I would roll back this change for the backport.

Reply:
I am unsure what happened here too... we discussed it a bit on slack. My leading theory is this is a devDependency quirk?

Running the following commands causes the lock file change:
```
$ cd dynamic-demo-plugin
$ rm -rf node_modules/ && yarn install && yarn build && yarn http-server
```